common/djangoapps/student/urls.py

```diff
@@ -45,6 +45,7 @@
 if settings.FEATURES.get('AUTOMATIC_AUTH_FOR_TESTING'):
     urlpatterns += (
         url(r'^auto_auth$', 'auto_auth'),
+        url(r'^update_user$', 'update_user'),
```

Comment:
That can be managed in `auto_auth`, I have done as this to just keep separate things separate. If we want to do it in `auto_auth` let me know.